### DISCUSSION TITLE
When playing a converted beatmap, warn that it is not a normal beatmap

### DISCUSSION THREAD

**Discussion by a3be3** [ppy/osu#15146] [Ideas]
For people who are new to osu, I think the converted beatmaps are misleading.

By playing converted beatmaps, those people might get fed up and quit this game.

And, for people who play any of the three modes of osu taiko ctb mania, converted beatmaps are very confusing.

So I think we need to show warnings like Loved beatmaps when we play converted beatmaps.
(The image when we start playing loved beatmaps is pasted under this sentence.)

I don't have any programming knowledge, so I can only give my opinion, but I hope you can fix it if you want.

Translated by DeepL

![screenshot1438](https://user-images.githubusercontent.com/92656078/137606716-48379e54-46e6-4bbf-a80a-102daa5df19d.png)

**Comment by peppy** [comment]
We definitely don't want a fullscreen banner like this, but the new player loading screen was made to show information like this. Whether we show a convert using an icon or a message is up for consideration, but it should definitely be part of the information which is presented to the player before they start playing.

**Reply by RocketSheepOriginal** [reply]
This warning message could be shown in the loading screen, the screen where you can change blur, beatmap sounds, etc.

**Reply by peppy** [reply]
Yes, that is precisely what I meant.
